### ISSUE TITLE
cljs implementation

### ISSUE THREAD

**Issue by verberktstan** [sdedovic/easings-clj#1]
I think we could implemant this with js math as well, would be nice to have it working both in JS and Java.

Do you mind me taking a look? I'll file a small PR if i manage to improve it :-)

**Comment by verberktstan** [comment]
For example this works fine:

```
(defn ease-in-sine [t]
  (- 1 (-> t (* js/Math.PI) (/ 2) js/Math.cos)))
```

And we could move this to easings/core.cljcand make it work in both java & js like so:
```
(defn ease-in-sine [t]
(let [pi #?(:cljs js/Math.PI :clj Math.PI)
      cos #?(:cljs js/Math.cos :clj Math.cos)]
  (- 1 (-> t (* pi) (/ 2) cos))))
```

If we extract the pi & cos bindings out of this function, these can be used in all easing functions.

I'll see if i can make a fork and let you know!

**Comment by sdedovic** [comment]
I took a slightly different approach, following this:
https://github.com/MastodonC/kixi.stats/blob/master/src/kixi/stats/math.cljc

PR is open #3

**Comment by verberktstan** [comment]
Cool, that looks great!

**Comment by sdedovic** [comment]
Merged #3 and released to [Clojars](https://clojars.org/com.dedovic/easings-clj/versions/0.2.0)
